datafusion/common/src/config.rs

```diff
@@ -181,6 +224,119 @@ macro_rules! config_namespace {
     }
 }
 
+config_namespace! {
+    pub struct ConfigFileDecryptionProperties {
+        pub footer_key_as_hex: String, default = String::new()
+        pub column_keys_as_json_hex: String, default = String::new()
+        pub aad_prefix_as_hex: String, default = String::new()
+    }
+}
```

Comment:
This is a pure string based config that we can add directly to the datafusion configuration. This allows use from the CI and follows the conventions of the other parameters.